### DISCUSSION TITLE
Any idea about when v0.32 will be released?

### DISCUSSION THREAD

**Discussion by fredpan** [thanos-io/thanos#6488] [General]
Hi all,

Thanos doc states that Thanos will have minor releases every 6 weeks. But it's been a while since Thanos did not have a  newer release. 

There are a bunch of new features/bug fixes I need. Any idea about when will v0.32 be released?

**Comment by GiedriusS** [comment]
I think the informal consensus is that https://github.com/thanos-io/thanos/issues/6257 needs to be fixed first.
